### ISSUE TITLE
No Fonts desplayed on iOS with the Microsoft RDP App

### ISSUE THREAD

**Issue by poldixd** [neutrinolabs/xrdp#367]
Hey,

I use the Microsoft RDP App 8.1.20 in an iPad 4 Mini and iPhone 6 on the latest iOS Version. My xrdp Version is 0.5.0-2 on Debian 7.10 (raspberry Pi 2).

If i connect to my raspberry Pi, the fonts will not displayed. 

![image](https://cloud.githubusercontent.com/assets/695449/14767165/ad8db3f0-0a1f-11e6-8678-dc3cff46a6eb.png)
![image](https://cloud.githubusercontent.com/assets/695449/14767166/addd160c-0a1f-11e6-89a4-f028415aece5.png)

Its the same on my iPad and my iPhone.

Here are my configs:
sesman.ini

```
[Globals]
ListenAddress=127.0.0.1
ListenPort=3350
EnableUserWindowManager=1
UserWindowManager=startwm.sh
DefaultWindowManager=startwm.sh

[Security]
AllowRootLogin=1
MaxLoginRetry=4
TerminalServerUsers=tsusers
TerminalServerAdmins=tsadmins

[Sessions]
MaxSessions=3
KillDisconnected=0
IdleTimeLimit=0
DisconnectedTimeLimit=0

[Logging]
LogFile=/var/log/xrdp-sesman.log
LogLevel=DEBUG
EnableSyslog=0
SyslogLevel=DEBUG

[X11rdp]
param1=-bs
param2=-ac
param3=-nolisten
param4=tcp

[Xvnc]
param1=-bs
param2=-ac
param3=-nolisten
param4=tcp

```

xrdp.ini

```
[globals]
bitmap_cache=yes
bitmap_compression=yes
port=3389
crypt_level=high
channel_code=1

[xrdp1]
name=Verbindung NEU
lib=libvnc.so
username=XXX
password=ask
ip=127.0.0.1
port=-1

[xrdp2]
name=Verbindung wiederherstellen
lib=libvnc.so
username=XXX
password=ask
ip=127.0.0.1
port=5910
```

**Comment by metalefty** [comment]
Using such a too old version is no longer reasonable.
However, I have the same issue in current devel branch (0.9.0beta)


![2016-04-25 10 40 05](https://cloud.githubusercontent.com/assets/941609/14771938/964f7fbc-0ad2-11e6-83e4-8d9b9bbb093e.png)
![2016-04-25 10 39 56](https://cloud.githubusercontent.com/assets/941609/14771939/9652f3e0-0ad2-11e6-8437-a702b1f1d2a2.png)

**Comment by mehmeteminkartal** [comment]
Same with my raspberry pi 3
![img_0446](https://cloud.githubusercontent.com/assets/5096916/16541615/de2aa274-408f-11e6-9792-186e854768dc.PNG)

**Comment by NANO-Chile** [comment]
how can I fix this please? Im using Jump Desktop in OSX and getting same problem.

**Comment by necktwi** [comment]
Its 2018 N I bumped into it

**Comment by FaimMedia** [comment]
Also experiencing this problem...

**Comment by metalefty** [comment]
@poldixd @mehmeteminkartal @NANO-Chile @necktwi @FaimMedia 
Everyone, the issue is fixed by #1235. Will be included next v0.9.9 release.
